iidm/iidm-converter-api/src/main/java/com/powsybl/iidm/export/Exporter.java

```diff
@@ -43,4 +55,25 @@
      */
     void export(Network network, Properties parameters, DataSource dataSource);
 
+    /**
+     * Conversion from iidm format.
+     *
+     * @param network the iidm model
+     * @param parameters some properties to configure the export
+     * @return the native data model
+     */
+    default Object conversion(Network network, Properties parameters) {
```

Comment:
Splitting the export into network -> nativeDataModel then nativeDataModel -> file is probably not a good idea as we can imagine in some cases no need to intermediate native data model. Also for most of the use cases we don't care about of the native data model and we don't want to see it in the export interface.
I would simply return it in the orginal export method and everybody can ignore it (and also it can be null)

Reply:
If we did like this, the export method generate the export file and return the nativeDataModel witch can be use by the postprocessor, but the postprocessor have to export another file (or override).
Is it really what we expect ?

Reply:
You just have to pass the post processors to the export method and let the implementation call them (or not, just do it for UCTE exporter) at the right place in the export code.